CapitalGainsImporter.js

```diff
@@ -139,12 +139,15 @@ CKI.inputRowData = function(dataSource, row) {
 			var formIndex = rowResp.formIndex;
 			var rowIndex = rowResp.rowIndex;
 
-			var ht = element.querySelector('[name="capitalGains['+formIndex+'].holdingType"]');
-			ht.value = data.holdingType;
-
-			var rc = element.querySelector('[name="capitalGains['+formIndex+'].reported"]');
+			var rc = element.querySelector('[name="capitalGains['+formIndex+'].reportingCategory"]');
```

Comment:
CK also has a "belongsTo" column before this (as of this writing).
Possible values of the dropdown are:
tp  (Primary tax filer), sp (Spouse), or bt (Both)